css/css-text/word-break/word-break-auto-001.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title>word-break: auto</title>
+<meta name="assert" content="word-break: auto keeps phrases together.">
+<link rel='help' href='https://drafts.csswg.org/css-text-3/#word-break-property'>
+<link rel='match' href='reference/word-break-auto-ref-001.html'>
```

Comment:
nit: we usually name these 001-ref.html as opposed to ref-001.html fwiw.
ditto for other files.